integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonMergerRDD.scala

```diff
@@ -91,6 +91,7 @@ class CarbonMergerRDD[K, V](
   var singleRange = false
   var expressionMapForRangeCol: util.Map[Integer, Expression] = null
   var broadCastSplits: Broadcast[CarbonInputSplitWrapper] = null
+  val bucketInfo = carbonLoadModel.getCarbonDataLoadSchema.getCarbonTable.getBucketingInfo
```

Comment:
Seems executor side does not need this variable, so move it into internalCompute to make it as a local variable